accounting/invoice/templates/default/default.html

```diff
@@ -0,0 +1,134 @@
+                  {% if user.address.state %}
+                    <span class="user-state">{{ user.address.state }}</span>,
+                  {% endif %}
+                  <span class="user-zipcode">{{ user.address.zipcode }}</span><br>
+                  <span class="user-country">{{ user.address.country.name }}</span>
+                  {% if user.vat %}
+                    <br><span class="user-vat">VAT number: {{ user.vat }}</span>
+                  {% endif %}
+                </div>
+              </div>
+            {% endfor %}
+          </div>
+          <div class="row">
+            <div id="client-bank-account-details" class="col-xs-6">
+              <h5 class="client-bank-account-header"><strong>BANK ACCOUNT</strong></h5>
+              Bank Name: {{ bank_account.bank.name }}<br>
+              Bank Account Type: {{ bank_account.type|title }}<br>
+              Bank Account Currency: {{ bank_account.currency }}<br>
+              {% for name, value in bank_account_details %}
+                {{ name }}: {{ value }}<br>
+              {% endfor %}
+            </div>
+            {% if invoice.extra_text %}
+            <div id="extra" class="col-xs-6">
+              <h5 class="extra-header"><strong>EXTRA</strong></h5>
+              <p id="extra-text">{{ invoice.extra_text|linebreaks }}</p>
+            </div>
+            {% endif %}
+          </div>
+        </header>
+
+        <main>
+          <table id="line-items" class="table table-striped table-condensed" cellpadding="0" cellspacing="0">
+            <tr>
+              <th scope="col" class="column column-key">Key</th>
+              <th scope="col" class="column column-name">Name</th>
+              <th scope="col" class="column column-quantity">Quantity</th>
+              <th scope="col" class="column column-price">Price</th>
+              <th scope="col" class="column column-total">Total</th>
+            </tr>
+
+            <tbody>
+              {% for line_item in line_items %}
+                <tr class="item">
+                  <th scope="row" class="column column-key">{{ line_item.key }}</th>
+                  <td class="column column-name">{{ line_item.name }}</td>
+                  <td class="column column-quantity">{{ line_item.quantity|floatformat:2 }}</td>
+                  <td class="column column-price">{{ line_item.price }} {{ currency }}</td>
+                  <td class="column column-total">{{ line_item.total|floatformat:2 }} {{ currency }}</td>
+                </tr>
+              {% endfor %}
+            </tbody>
+
+            <tr class="total">
+              <td colspan="2" class="column column-key column-name"></td>
+              <td class="column column-quantity">{{ total_quantity|floatformat:2 }}</td>
+              <td class="column column-price"></td>
+              <td class="column column-total">{{ total_cost|floatformat:2 }} {{ currency }}</td>
+            </tr>
```

Comment:
@UmanShahzad Could this row go inside `tbody` above?

Reply:
I'd say this is the footer, so it would go in `tfoot`. But then it suffers from the issue I mentioned above about `thead`, except this time by appearing on the bottom of each page.

Reply:
@UmanShahzad Haha, OK :)